src/python_testing/TC_LVL_2_3.py

```diff
@@ -90,16 +93,28 @@ async def test_TC_LVL_2_3(self):
 
         self.step(2)
         feature_map = await self.read_single_attribute_check_success(cluster=lvl, attribute=lvl.Attributes.FeatureMap)
+        attributes = await self.read_single_attribute_check_success(cluster=lvl, attribute=lvl.Attributes.AttributeList)
 
-        self.step(3)
-        max_level = await self.read_single_attribute_check_success(cluster=lvl, attribute=lvl.Attributes.MaxLevel)
+        self.step("3a")
+        attributes = await self.read_single_attribute_check_success(cluster=lvl, attribute=lvl.Attributes.AttributeList)
+
+        if lvl.Attributes.MaxLevel.attribute_id in attributes:
+            max_level = await self.read_single_attribute_check_success(cluster=lvl, attribute=lvl.Attributes.MaxLevel)
+        else:
+            max_level = 254
+
+        self.step("3b")
+        if lvl.Attributes.MinLevel.attribute_id in attributes:
+            min_level = await self.read_single_attribute_check_success(cluster=lvl, attribute=lvl.Attributes.MinLevel)
+        else:
+            min_level = 1
 
         self.step(4)
-        cmd = Clusters.LevelControl.Commands.MoveWithOnOff(moveMode=lvl.Enums.MoveModeEnum.kDown)
+        cmd = Clusters.LevelControl.Commands.MoveToLevelWithOnOff(level=min_level, transitionTime=0)
```

Comment:
you could use 0 instead of min level. it will move to the min level of the device regardeless

Reply:
Discussed on slack - going to leave this as is for now. It's fairly clear what's happening to someone that doesn't know how the cluster works and both work essentially the same.